src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -248,6 +248,13 @@ public void start(Future<Void> startFuture) {
       .create()
       .register(router);
 
+    new FakeStorageModuleBuilder()
+      .withRecordName("scheduled notice")
+      .withCollectionPropertyName("schedulednotices")
```

Comment:
This feedback might be in the wrong pull request, can this property be capitalised please `scheduledNotices`

Do I need to provide this feedback separately on the storage pull request?

Reply:
I'll do this change. But I think I miss something. Could you please explain how this value is related to storage module?

Reply:
The collection property name refers to the name of the array property that records are in when doing a get on the collection e.g. for /item-storage/items, the item records are in a property called `items`

It is important that these line up with the storage module interfaces, otherwise integration will likely fail.

Before the calendar API integration, which uses fakes that cannot be replaced with the real implementation easily, it was possible to run the mod-circulation API tests with either the fakes or the real modules, to detect discrepancies like this. However, that no longer works, because the tests do no set up the calendar sufficiently.

Does that make sense?

Reply:
I just checked, the [storage API](https://github.com/folio-org/mod-circulation-storage/pull/169/files#diff-efed83440ad9997c05dd1e4e87eb5903R6) uses the correct capitalisation.

This discrepancy could have caused run time failures when the modules are integrated.

Reply:
Thanks, now makes sense to me. So there is no necessity to point this in circulation-storage pull request. This property is capitalized there.

Reply:
Yes, this was a discrepancy between the fake (I imagine there is another one in the production code for creating the client) and the storage interface, which would've likely become apparent when we tried to integrate this work